test/xskbench/xskbench_common.h

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <xdpapi.h>
+#include "afxdp_helper.h"
```

Comment:
same as elsewhere, `""` includes should be `<>` unless they're in the current directory